pywbem/cim_obj.py

```diff
@@ -1702,13 +1706,20 @@ def property_list(self):
 
         This attribute is settable. For details, see the description of the
         same-named constructor parameter.
+
+        **Deprecated:** This attribute has been deprecated in pywbem v0.12.0.
+        Set only the desired properties on the object, instead of working with
+        this property filter.
         """
         return self._property_list
 
     @property_list.setter
     def property_list(self, property_list):
         """Setter method; for a description see the getter method."""
         if property_list is not None:
+            warnings.warn("The 'property_list' init parameter and attribute "
```

Comment:
I attached a note to the dependent pr.  Should we note that the correct way to use a property list to control modify wbem operations is through the property list on the operation???

Reply:
No. That is too far away, here we are in context of constructing a CIMInstance object. It may not even be the result of an Enumerate operation.